src/components/CovMap.tsx

```diff
@@ -216,17 +220,43 @@ export const CovMap = () => {
     }
   };
 
-  const flyToHome = () => {
-    // check for "valid" postal codes
-    if (!userPostalCode || isNaN(userPostalCode)) return;
-    dispatch(switchViewToPlace(String(userPostalCode)));
-  };
+  /**
+   * Takes the stored last select feature and sets it to current feature,
+   * to show the feature info last viewed after a reload.
+   */
+  useEffect(() => {
+    if (jumpedToLastFeatureOnce) {
+      return;
+    }
+
+    if (mapRef.current && currentVisual && mappedSets[currentVisual] && currentFeaturePropId) {
+      const mapset = mappedSets[currentVisual][currentFeaturePropId.mappingId];
+      if (mapset) {
+        const featurePropKey = config.visuals[currentVisual].mappings[currentFeaturePropId.mappingId].featurePropKey;
+        const feature = (mapset.geo as FeatureCollection).features.find(
+          ({ properties }) => (properties || {})[featurePropKey] === currentFeaturePropId.featurePropId,
+        );
+        const map = mapRef.current.getMap();
+
+        map.once("idle", (evt) => {
+          dispatch(
+            AppApi.setCurrentFeature(
```

Comment:
Fehlt hier nicht auch noch ein AppApi.setViewport?

Wenn ich in der momentanen Version München auswähle, dann nach Berlin scroll ohne einen neuen Landkreis zu wählen und dann die App neu lade, dann ist immer noch München ausgewählt, obwohl ich es nicht mehr auf dem Ausschnitt sehe (ich sehe dann Berlin auf der Karte).

Alternative: beim Verschieden der Karte das aktuell Feature löschen / Infobox ausblenden.

Reply:
Wuerde eher ein fly to zum letzten feature machen dann... Beim verschieben deselektieren klingt irgendwie als koennte es verwirren und sich komisch verhalten.

Reply:
Sollen wir den PR bereits ohne das "fly to last feature" mergen, oder erst wenn es konsistent ist? Der Rest passt von mir aus.